lib/node_modules/@stdlib/assert/is-same-date-object/benchmark/benchmark.js

```diff
@@ -0,0 +1,60 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isBoolean = require( '@stdlib/assert/is-boolean' ).isPrimitive;
+var pkg = require( '@stdlib/assert/is-same-date-object/package.json' ).name;
+var isSameDateObject = require( '@stdlib/assert/is-same-date-object/lib' );
+
+
+// MAIN //
+
+bench( pkg + '', function benchmark( b ) {
```

Comment:
```suggestion
bench( pkg, function benchmark( b ) {
```

No reason to append an empty string here.

Reply:
yes , about that , i meant to ask should we add some string here , - benchmark are being tested or smething like that , left an empty string here as a reminder